src/Ui/Button.elm

```diff
@@ -241,31 +248,62 @@ buttonText =
         }
 
 
-button : List Css.Style -> List (A11y.Attribute msg) -> Maybe msg -> ButtonContent -> A11y.Html msg
-button customStyle attrs mOnClick buttonContent =
+{-| -}
+account : { picture : String, name : String, onClick : msg } -> A11y.Html msg
```

Comment:
isn't this a profile selector or something? An "account" doesn't sound liek a ui component

Reply:
It's not really a profile selector since it's going to be more generic than that. Actually let's skip this one for now. I believe this is an image button.